tests/specifications/general_test.py

```diff
@@ -144,15 +144,24 @@ def NOT_IMPLEMENTED_test_check_039():
   # - FAIL, "Hints should NOT overlap!"
 
 
-def NOT_IMPLEMENTED_test_check_046():
+def test_check_046():
   """ Font contains the first few mandatory glyphs (.null or NULL, CR and
   space)? """
-  # from fontbakery.specifications.general import com_google_fonts_check_046 as check
-  # TODO: Implement-me!
-  #
-  # code-paths:
-  # - PASS, "Font contains the first few mandatory glyphs" 
-  # - WARN, "Font is missing glyphs for some mandatory codepoints"
+  from fontbakery.specifications.general import com_google_fonts_check_046 as check
+
+  test_font = TTFont(os.path.join("data", "test", "Nunito", "Nunito-Regular.ttf"))
```

Comment:
here this should work as well, and is a bit more readable:

```
test_font = TTFont("data/test/Nunito/Nunito-Regular.ttf")
```

Reply:
Please make this change. I think I saw something similar in one of your past commits as well. If you could find that it would be great to change as well, so that the codebase is kept with a uniform code-style.

Reply:
Also, the correct path for that font file should have the `nunito` directory name written in all lowercase.

Reply:
Using `os.path.join` means it will work cross platform. Win uses \ in paths.

Reply:
There are a few cases where you get away with using Unix-style paths in Python on Windows, but sometimes stuff gets mixed up and breaks. The better solution would be to use join across the codebase, even better if we required Python 3.6+ and use pathlib.Path for everything :)